tools/ci_build/github/azure-pipelines/android-x86_64-crosscompile-ci-pipeline.yml

```diff
@@ -261,6 +261,7 @@ jobs:
         --android_abi=x86_64 \
         --android_api=29 \
         --use_nnapi \
+        --build_shared_lib \
         --test
```

Comment:
Out of interest why was this change required?

Reply:
This change is only affect a shared library, so the test is attached to onnxruntime_shared_libs_test, so this is enabled.

Reply:
This line is reverted after a rebase and squash.